insertion_sort.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import unicode_literals
+from time import time
+
+
+def insertion_sort(list):
+    for index in range(1, len(list)):
+        value = list[index]
+        i = index - 1
+        while i >= 0:
+            if value < list[i]:
+                list[i + 1] = list[i]
+                list[i] = value
+                i = i - 1
+            else:
+                break
+
+
+if __name__ == '__main__':
+    def worst_case():
+        list_a = range(100)
+        list_a.reverse()
+        return list_a
+
+    def best_case():
+        list_b = range(100)
+        return list_b
+
```

Comment:
There needs to be a variety of list lengths here. From the assignment description: "demonstrates the performance characteristics of this sort _over a variety of lengths of input_ in both the best and worst-case scenarios."